src/frontend/components/Felleskomponenter/Pensjonsmodal/språkUtils.ts

```diff
@@ -57,8 +59,26 @@ export const pensjonFlerePerioderSpmSpråkId = (
             : 'eøs-om-deg.leggtilpensjon.spm';
 };
 
-export const mottarEllerMottattPensjonSpråkId = () =>
-    dinLivssituasjonSpørsmålSpråkId[DinLivssituasjonSpørsmålId.mottarUtenlandspensjon];
+export const mottarEllerMottattPensjonSpråkId = (
+    gjelderUtlandet: boolean,
+    gjelderAndreForelder: boolean,
+    andreForelderErDød: boolean
+): string => {
+    if (gjelderAndreForelder) {
+        if (andreForelderErDød) {
+            return gjelderUtlandet
+                ? omBarnetSpørsmålSpråkId[OmBarnetSpørsmålsId.andreForelderPensjonUtlandEnke]
+                : eøsBarnSpørsmålSpråkId[EøsBarnSpørsmålId.andreForelderPensjonNorgeEnke];
+        } else {
+            return gjelderUtlandet
+                ? omBarnetSpørsmålSpråkId[OmBarnetSpørsmålsId.andreForelderPensjonUtland]
+                : eøsBarnSpørsmålSpråkId[EøsBarnSpørsmålId.andreForelderPensjonNorge];
+        }
+    } else {
+        return dinLivssituasjonSpørsmålSpråkId[DinLivssituasjonSpørsmålId.mottarUtenlandspensjon];
+        //TODO: return gjelderUtlandet ? dinLivssituasjonSpørsmålSpråkId[DinLivssituasjonSpørsmålId.mottarUtenlandspensjon] : eøsSøkerSpørsmålSpråkId[EøsSøkerSpørsmålsId.pensjonNorge];
+    }
```

Comment:
trenger ikke å gjøres nå ?

Reply:
Finnes ikke EøsSøkerSpørsmålsId  enda, men ville ikke få for mye konflikter med oppgaven til Nora ved å opprette en, så da tenkte jeg å legge inn denne etter oppgaven hennes 😊